robocop/run.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, from_cli=False, config=None):
         if not from_cli:
             self.config.reports.add('json_report')
         self.out = self.set_output()
+        if from_cli:
+            print("### DEPRECATION WARNING: The rule 0906 (redundant-equal-sign) is "
```

Comment:
Maybe we should suggest excluding this rule? So people would be prepared for its removal.